### PR TITLE
3150 log target host ip

### LINKED ISSUE #3150: Log only TargetHost's IP
## Description

Our logs are loaded with extraneous data. Specifically, we're logging the entire contents of `TargetHost` in multiple places. This makes it hard to read the logs.

Log only `TargetHost.ip` in our debug messages

## Tasks
- [ ] Fix it. (0.25d)

### PR DESCRIPTION
**Pull request by ordabach**
# What does this PR do?

Fixes #3150.

Instead of logging the whole TragetHost object, it logs only the host IP.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Do all unit tests pass?
* [x] Do all end-to-end tests pass?
* [x] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

### CODE REVIEW

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Missed some statements in:
`monkey/infection_monkey/exploit/mssqlexec.py:72`
`monkey/infection_monkey/exploit/mssqlexec.py:88`
`puppet/puppet.py:88`
A bunch of in:
`monkey/infection_monkey/exploit/sshexec.py`
A bunch in:
`monkey/infection_monkey/exploit/wmiexec.py`
`monkey/infection_monkey/exploit/zerologon.py`

I'm mainly worried about failing exploiters (pretty common) spamming the logs with the same kind of information (open services didn't change based on exploiter, so it doesn't make sense to log them for each host)

**Review by mssalvatore — APPROVED** [review]

**Review by mssalvatore — CHANGES_REQUESTED** [review]
@ordabach Flake8 and Black fail. Do you have pre-commit hooks installed?

File: monkey/infection_monkey/exploit/mssqlexec.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def _brute_force(
 
         logger.warning(
             "No user/password combo was able to connect to host: {0}:{1}, "
-            "aborting brute force".format(host, port)
+            "aborting brute force".format(host.ip, port)
```
Comment (VakarisZ) [inline monkey/infection_monkey/exploit/mssqlexec.py:156]: `host` is already an IP... this is what happens when people neglect good naming conventions

**Review by VakarisZ — APPROVED** [review]

**Review by ilija-lazoroski — APPROVED** [review]
All good with testing, just squash the commits with the same message.

**Pull request merged**